src/components/organisms/Badges/Badges.tsx

```diff
@@ -85,6 +87,22 @@ export const Badges: React.FC<{
       .filter((b) => b !== undefined);
   }, [relevantVisits, venues, currentVenue, relatedVenueIds]);
 
+  const badgeList = useMemo(
+    () =>
+      badges.filter(notEmpty).map((b) => (
+        <li className="badge-list-item" key={b.label}>
+          <Link to={getLocationLink(b.venue, b.room)}>
+            <img
+              className="badge-list-item-image"
+              src={b.image}
+              alt={`${b.label} badge`}
+            />
```

Comment:
[nit] I generally prefer to use full names for variables, reduces cognitive complexity when reading. So in this instance, it would be using `badge` instead of `b`